pkg/app/piped/cloudprovider/lambda/client.go

```diff
@@ -29,7 +30,13 @@ import (
 	"go.uber.org/zap"
 )
 
-const defaultAliasName = "Service"
+const (
+	defaultAliasName = "Service"
+	// RequestRetryTime represents the number of times calling to AWS resource control.
+	RequestRetryTime = 3
+	// RetryIntervalDuration represents duration time between retry.
+	RetryIntervalDuration = 1 * time.Minute
```

Comment:
How about moving these into the `executor/lambda` package as private consts?

Reply:
Since it an attribute attached to the AWS API, I think it would be better if it were placed in the AWS client wrapper package.

Otherwise, I planned to use those constant here to [UpdateFunction src](https://github.com/pipe-cd/pipe/blob/master/pkg/app/piped/cloudprovider/lambda/client.go#L153)

Reply:
> Since it's an attribute attached to the AWS API

I am not sure they belong to AWS API or the API callers.

But just a nit. 👍